src/Command/TwoStepVerificationCommand.php

```diff
@@ -71,8 +69,9 @@ public function configure(): void
     /**
      * {@inheritdoc}
      */
-    public function execute(InputInterface $input, OutputInterface $output): void
+    public function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
Changing the return type is a BC break for whose are extending this method. IMO, removing the return type we can respect the BC promise.
```suggestion
    public function execute(InputInterface $input, OutputInterface $output)
```

Reply:
"or allowing execute() method to return integer values"

but to allow `execute` return  `int`, I need to change this return type!

Reply:
Sorry I see in your suggestion, that you're removing the type-hint.